### PR TITLE
[1LP][RFR] test_events.py fix

### PR DESCRIPTION
**Pull request by izapolsk**
Purpose or Intent
=================
 test_events.py fix

{{pytest: --use-provider=vsphere55 cfme/tests/cloud_infra_common/test_events.py}}

### CODE REVIEW

**Comment by RonnyPfannschmidt** [comment]
@mshriver can we pick better names for `true`/`false` they are painfully meaningless and it seems like they lead to mistakes in using the variables

**Comment by izapolsk** [comment]
@RonnyPfannschmidt, I absolutely agree with you on that.
I'll fix this here but this code was recently developed by @quarckster. So, I'd ask him not to use such names in future.

**Comment by RonnyPfannschmidt** [comment]
@izapolsk oh, i see, i should have taken a look at who did the actual code, sorry

File: cfme/control/explorer/policies.py [hunk 1]
```diff
@@ -376,21 +376,21 @@ def assign_actions_to_event(self, event, actions):
                 TRUE section. If :py:class:`dict`, the action is key and value specifies its
                 placement. If it's True, then it will be put in the TRUE section and so on.
         """
-        true, false = [], []
```
Comment (quarckster) [inline cfme/control/explorer/policies.py]: Actually these "true, false"  were not invented by me https://github.com/ManageIQ/integration_tests/blame/7c2dce82524be9598e45e42f820447d66febd681/cfme/control/explorer.py#L752, but I admit blind copy/paste is bad.

**Comment by izapolsk** [comment]
there are quite different issues and their investigation takes time. So, I've decided to fix the tests each in turn.

**Pull request merged**